asv_bench/benchmarks/arithmetic.py

```diff
@@ -413,17 +413,6 @@ def setup(self):
     def time_add_overflow_arr_rev(self):
         checked_add_with_arr(self.arr, self.arr_rev)
 
-    def time_add_overflow_arr_mask_nan(self):
```

Comment:
Are these considered superfluous?

Reply:
they use a no-longer-used mask argument.  actually these are all pretty superfluous.  will update to remove